design/example.go

```diff
@@ -6,6 +6,7 @@ import (
 	"regexp"
 	"time"
 
+	"github.com/goadesign/goa"
```

Comment:
This import is not OK, the design package shouldn't rely on the goa package - it should be the opposite, thanks!

Reply:
OK, I'll fix it.